### PR TITLE
Revert "Revert "Revert "Revert "Revert "Revert "Revert "Revert "Test noti""""""""

### PR DESCRIPTION
**Pull request by chipset508**
Reverts chipset508/CodeschoolPrework#9

### CODE REVIEW

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

**Comment by chipset508** [comment]
a

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def index
     if params[:search]
       @articles = Article.search(params[:search])
     else
+      puts 'Test noti'
```
Comment (chipset222) [inline app/controllers/articles_controller.rb]: a

**Pull request merged**